web/source/kmwembedded.ts

```diff
@@ -610,30 +612,30 @@ namespace com.keyman.osk {
    *  @return {boolean}         true if key code successfully processed
    */
   keymanweb.processDefaultMapping = function(code, shift, Lelem, keyName) {
-    let VisualKeyboard = com.keyman.osk.VisualKeyboard;
-    if (code == VisualKeyboard.keyCodes.K_SPACE) {
+    let Codes = com.keyman.text.Codes;
+    if (code == Codes.keyCodes.K_SPACE) {
       kbdInterface.output(0, Lelem, ' ');
       return true;
-    } else if (code == VisualKeyboard.keyCodes.K_ENTER) {
+    } else if (code == Codes.keyCodes.K_ENTER) {
       kbdInterface.output(0, Lelem, '\n');
       return true;
-    } else if (code == VisualKeyboard.keyCodes.K_TAB) {
+    } else if (code == Codes.keyCodes.K_TAB) {
```

Comment:
Note, in #1526 K_ENTER and K_TAB are removed here for the native apps to dispatch the keys.

Reply:
Won't these be picked up when we merge that beta fix into master? (FWIW, it's probably helpful to make sure this doesn't change behaviour on desktop!)